docs/consume-events.md

```diff
@@ -160,55 +160,89 @@ See for instance this Java code that consumes messages from a topic named “tes
-                lifecycle: the component's lifecycle, eg. development or production
-                owner: product team in charge of consuming component
-                subscribesToEvent: [event-name, event-name-two]
-        ```
-
-    ??? System Example
-        ``` { .yaml .copy }
-            apiVersion: backstage.io/v1alpha1
-            kind: System
-            metadata:
-                name: your system name
-                description: what the system does
-            spec:
-                owner: product team in charge of consuming system
-                domain: domain the system falls under (optional)
-                subscribesToEvent: [event-name, event-name-two]
-        ```
-
-3. If you are unsure whether to classify your consumer as a Component or a System, see the [Backstage System Model](https://backstage.io/docs/features/software-catalog/system-model/).
-
-4. Once your `catalog-info.yaml` file has been committed, log into [CODE VA](https://code.va.gov/) while on the VA network, and follow the [default Backstage provided method](https://backstage.io/docs/features/software-catalog/#adding-components-to-the-catalog) for adding entries to the catalog.
-
-**NOTE**: As a consumer, it is imperative that you include the `subscribesToEvent` in your `catalog-info.yaml` file, in the `spec` object. `subscribesToEvent` is an array containing strings. Each string corresponds to a `name` specified in a producer's `catalog-info.yaml` file [metadata object](https://backstage.io/docs/features/software-catalog/descriptor-format#common-to-all-kinds-the-metadata). This metadata name property can not always be derived from the event or the topic, so it will require referencing the producer's `catalog-info.yaml` file, e.g.:
+1. In CODE VA, an event-consuming software entity can be modeled as a [Component](https://backstage.io/docs/features/software-catalog/descriptor-format/#kind-component) or as a [System](https://backstage.io/docs/features/software-catalog/descriptor-format#kind-system). If you are unsure whether to classify your consumer as a Component or a System, see the [Backstage System Model](https://backstage.io/docs/features/software-catalog/system-model/).
 
+2. Create a file named `catalog-info.yaml` at the root of your source code repository and populate it with the applicable template, updating `metadata` and `spec` with values that correspond to your component or system.
+
+3. Once your `catalog-info.yaml` file has been committed it will be automatically picked up after some time and the software entity will be viewable on [CODE VA](https://code.va.gov/) (must be on the VA network to view). If you would like the event to display quicker, log into [CODE VA](https://code.va.gov/) while on the VA network and follow the [default Backstage provided method](https://backstage.io/docs/features/software-catalog/#adding-components-to-the-catalog) for adding entries to the catalog.
```

Comment:
Can we be more precise than "after some time"?

Reply:
I can follow up with the CODE VA team about this on Monday